src/app/map/[id]/Tile/auth.dynamic.tsx

```diff
@@ -0,0 +1,40 @@
+"use client";
+
+import React, { useState } from "react";
+import { LoginForm } from "~/components/auth/LoginForm";
+import { RegisterForm } from "~/components/auth/RegisterForm";
+
+interface AuthTileProps {
+  initialView?: "login" | "register";
+}
+
+// This component will be dynamically imported
+export default function AuthTile({ initialView = "login" }: AuthTileProps) {
```

Comment:
This Tile should be tested with storybook stories. Please implement the static version (stateless) in auth.static.tsx and the stories in auth.static.stories.tsx